tasks/lib/packager-browserify.js

```diff
@@ -46,25 +37,13 @@ module.exports = function generate(platform, useWindowsLineEndings, platformVers
         outReleaseFile = path.join('pkg', 'cordova.' + platform + '.js');
         outReleaseFileStream = fs.createWriteStream(outReleaseFile);
 
-        var symbolList = null;
-        var addSymbolList = through.obj(function(row, enc, next) {
-            if(symbolList === null) {
-                symbolList = requireTr.getModules(platform);
-                this.push(util.format('var symbolList = %s;\n%s\n', JSON.stringify(symbolList, null, 4), row));
-            } else {
-                this.push(row);
-            }
-            next();
-        });
-
-        libraryRelease.pipeline.get('wrap').push(addSymbolList);
-
         // write license header
         writeLicenseHeader(outReleaseFileStream, platform, commitId, platformVersion);
 
-        releaseBundle = libraryRelease.bundle();
-
-        releaseBundle.pipe(outReleaseFileStream);
+        bundle(platform, false, commitId, platformVersion, platformPath)
+          .add(path.resolve(__dirname, '..', '..', 'src/scripts/bootstrap.js'))
```

Comment:
Yes, they both needed. This was moved from `bundle-browserify.js` direcly to `cordova-lib/prepare-browserify` due to some reasons (see https://github.com/apache/cordova-lib/pull/275#issuecomment-127782310) but we still need to add this file when building clean cordova-js bundle (without plugins) from grunt task.